File: GestionIntApi/DTO/TiendaDTO.cs
namespace GestionIntApi.DTO
{
    public class TiendaDTO
    {


        public int Id { get; set; }
        public string NombreTienda { get; set; }
        public string NombreEncargado { get; set; }
        public string Telefono { get; set; }
        public string Direccion { get; set; }

        public DateTime FechaRegistro { get; set; }

        public int ClienteId { get; set; }
    }
}

File: lib/models/tienda_dto.dart
class TiendaDTO {
  int id;
  String nombreTienda;
  String nombreEncargado;
  String telefono;
  String direccion;
  DateTime fechaRegistro;
  int clienteId;

  TiendaDTO({
    this.id = 0,
    required this.nombreTienda,
    required this.nombreEncargado,
    required this.telefono,
    required this.direccion,
    required this.fechaRegistro,
    this.clienteId = 0,
  });

  Map<String, dynamic> toJson() => {
    'Id': id,
    'NombreTienda': nombreTienda,
    'NombreEncargado': nombreEncargado,
    'Telefono': telefono,
    'Direccion': direccion,
    'FechaRegistro': fechaRegistro.toIso8601String(),
    'ClienteId': clienteId,
  };
}